src/status_im/contact/core.cljs

```diff
@@ -31,12 +31,6 @@
              (update :contacts/contacts #(merge contacts %))
              (assoc :contacts/blocked (contact.db/get-blocked-contacts all-contacts)))}))
 
-(defn can-add-to-contacts? [{:keys [pending? dapp?]}]
-  (and (not dapp?)
-       (or pending?
-           ;; it's not in the contact list at all
-           (nil? pending?))))
-
 (defn build-contact [{{:keys [chats] :account/keys [account]
                        :contacts/keys [contacts]} :db} public-key]
   (cond-> (assoc (contact.db/public-key->contact contacts public-key)
```

Comment:
we can probably remove the address thingy as it is computed already in public-key->contact function